lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_perform_step.jl

```diff
@@ -90,7 +102,49 @@ end
     @.. broadcast=false k₂+=k₁
     @.. broadcast=false u=uprev + dt * k₂
     stage_limiter!(u, integrator, p, t + dt)
-    step_limiter!(u, integrator, p, t + dt)
+
+    if cache isa Rosenbrock23Cache
+        step_limiter!(u, integrator, p, t + dt)
+    end
```

Comment:
I think this should be running on both integrators. I think it was just missed for Rosenbrock32 because people don't care about it.